Userland/Libraries/LibCrypto/Checksum/CRC32.h

```diff
@@ -12,10 +12,16 @@
 
 namespace Crypto::Checksum {
 
+namespace Polynomial {
+static constexpr u32 Ethernet = 0xEDB88320;
+static constexpr u32 Ogg = 0x04C11DB7;
```

Comment:
For what it's worth, the "more accurate" name for these would be `CRC32Reversed` and `CRC32Normal` (the `CRC32` being in contrast to `CRC32C`, `CRC32K`, etc.).

I'm not sure if we actually want to act on that, just for consideration.